schema/cohortAlleleFreq.yaml

```diff
@@ -254,17 +254,23 @@ properties:
         additionalProperties: false
       homozygotes:
         type: integer
-      meanDepth:
-        type: number
-      qcFilters:
-        type: array
-        items:
-          type: string
-      lowComplexityRegion:
-        type: boolean
-      heterozygousAlleleBalanceFlagged:
+      hemizygotes:
         type: integer
     additionalProperties: false
+  qualityMeasures:
```

Comment:
This doesn't include `additionalProperties: false`. Is that intentional? @ahwagner 

Also `type: object`

Reply:
Not intentional. Good catch @theferrit32